azure/microservice-ecommerce/blueprint.yaml

```diff
@@ -0,0 +1,201 @@
+    dependsOnTrue: ProvisionCluster
+
+  - name: StorageAccessKey
+    type: Input
+    description: "Enter the Storage Account key for accessing the storage (see README.md step 3.3):"
+    dependsOnTrue: ProvisionCluster
+
+  - name: StorageContainerName
+    type: Input
+    description: "Enter the Storage Container name (see README.md step 3.4):"
+    default: terraform-state
+    dependsOnTrue: ProvisionCluster
+  # ############################################################################
+
+  - name: ClusterName
+    type: Input
+    description: What name do you want to give the cluster?
+    pattern: "^[[:alpha:]][\\w-]{1,29}[\\w]$"
+    # dependsOnTrue: ProvisionCluster
+
+  - name: Namespace
+    type: Input
+    default: xl-demo
+
+  - name: XLDUrlForXLR
+    type: Input
+    description: XL Deploy URL for XL Release (This will only be used in the XL Release template)
+    default: http://xl-deploy:4516
+
+  # ############################################################################
+  # Jenkins
+  # ############################################################################
+  - name: enableCICD
+    type: Confirm
+    description: Do you want to enable CI/CD integration with Jenkins?
+
+  - name: StoreAdminUsername
+    type: Input
+    description: Store Admin username
+    value: admin
+
+  - name: StoreAdminPassword
+    type: Input
+    description: Store Admin password
+    value: admin
+    secret: true
+  # ############################################################################
+
+  files:
+  - path: xebialabs.yaml
+  - path: README.md
+  - path: xebialabs/xld-infra-env.yaml.tmpl
+  - path: xebialabs/xld-terraform-apps.yaml.tmpl
+  - path: xebialabs/xld-kubernetes-invoice-app.yaml.tmpl
+  - path: xebialabs/xld-kubernetes-store-app.yaml.tmpl
+  - path: xebialabs/xld-kubernetes-notification-app.yaml.tmpl
+  - path: xebialabs/xlr-pipeline-ci-cd.yaml.tmpl
+  - path: xebialabs/xlr-pipeline-destroy.yaml.tmpl
+  - path: xebialabs/USAGE.md.tmpl
+  # Terraform
+  - path: terraform/main.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/variables.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/backend.tf.tmpl
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/outputs.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/.gitignore
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/aks/main.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/aks/variables.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/aks/outputs.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/vpc/main.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/vpc/variables.tf
+    dependsOnTrue: ProvisionCluster
+  - path: terraform/vpc/outputs.tf
+    dependsOnTrue: ProvisionCluster
+  # docker-compose setup for required tools
+  - path: docker/docker-compose.yml.tmpl
+  - path: docker/data/configure-xl-devops-platform.yaml
+  - path: docker/jenkins/jenkins.yaml
```

Comment:
The jenkins file can be with a dependson for enableCICD

Reply:
Don